src/SignupRouter.js

```diff
@@ -18,6 +19,7 @@ const router = createSwitchNavigator(
     Signup,
     InvalidW3TokenError,
     SigninInfo,
+    IOSWebAppSignIn,
   },
   {
     initialRouteName: 'Auth',
```

Comment:
make IOSWebaAppSignIn default route in case of isIOS+isWebApp, if user is signed in he will not get SignupRouter but Router.js
check beta branch for the utility constant isWebApp i did

Reply:
done